modules/Bio/EnsEMBL/G2P/Utils/Downloads.pm

```diff
@@ -71,15 +74,17 @@ sub download_data {
   # confidence_category, allelic_requirement and mutation_consequence  
   $confidence_category_attribs = $attribute_adaptor->get_attribs_by_type('confidence_category');  
   $allelic_requirement_attribs = $attribute_adaptor->get_attribs_by_type('allelic_requirement');
+  $cross_cutting_modifier_attribs = $attribute_adaptor->get_attribs_by_type('cross_cutting_modifier');
   $mutation_consequence_attribs = $attribute_adaptor->get_attribs_by_type('mutation_consequence');
+  $mutation_consequence_flag_attribs = $attribute_adaptor->get_attribs_by_type('mutation_consequence_flag');
 
   my $dbh = $GFD_adaptor->dbc->db_handle;
 
   my $csv = Text::CSV->new ( { binary => 1, eol => "\r\n" } ) or die "Cannot use CSV: ".Text::CSV->error_diag ();
   open my $fh, ">:encoding(utf8)", "$file" or die "$file: $!";
 
   # Write header to file
-  $csv->print($fh, ['gene symbol', 'gene mim', 'disease name', 'disease mim', 'DDD category', 'allelic requirement', 'mutation consequence', 'phenotypes', 'organ specificity list', 'pmids', 'panel', 'prev symbols', 'hgnc id', 'gene disease pair entry date']);
+  $csv->print($fh, ['gene symbol', 'gene mim', 'disease name', 'disease mim', 'confidence category', 'allelic requirement', 'cross cutting modifier', 'mutation consequence', 'mutation consequence flag', 'phenotypes', 'organ specificity list', 'pmids', 'panel', 'prev symbols', 'hgnc id', 'gene disease pair entry date']);
```

Comment:
We are effectively adding a couple of new columns here. If we add them at the end, it will be simpler for people using the file to modify their systems.